### PR TITLE
update openapi.yaml

### LINKED ISSUE #2643: `core/openapi.yaml` is significantly outdated
The requests used as input for standalone simulation are outdated, in particular anything related to allowances

### PR DESCRIPTION
**Pull request by majaziri**

### CODE REVIEW

File: core/openapi.yaml [hunk 1]
```diff
@@ -262,24 +266,15 @@ components:
...
-              - KVB
+        a:
+          type: number
+          format: double
+        b:
+          type: number
+          format: double
+        c:
+          type: number
+          format: double
```
Comment (multun) [inline core/openapi.yaml]: I'm very confused, these fields should match what's in [the railjson schema](https://github.com/DGEXSolutions/osrd/blob/dev/api/osrd_infra/schemas/rolling_stock.py)

File: core/openapi.yaml [hunk 2]
```diff
@@ -605,6 +553,8 @@ components:
           minItems: 1
           items:
             $ref: "#/components/schemas/TrainSchedule"
+        trains_path:
+        ######## ask about it
```
Comment (multun) [inline core/openapi.yaml]: Here I am :)

Comment (majaziri) [reply core/openapi.yaml]: I did not find the corresponding class in the code to get the details of trains_path.
There is also a similar comment related to 'modes' in rollingStock

File: core/openapi.yaml (hunk 1 above)
Comment (majaziri) [reply core/openapi.yaml]: ok i will check it

File: core/openapi.yaml [hunk 3]
```diff
@@ -343,58 +343,41 @@ components:
           type: number
           format: double
           example: 0.25
-        timetable_gamma:
+        gamma:
           type: number
           format: double
           example: 0.5
-        tractive_effort_curve:
```
Comment (multun) [inline core/openapi.yaml:350]: while it is true that this field is not there anymore, it was replaced by `effort_curves`, which should be added

File: core/openapi.yaml [hunk 4]
```diff
@@ -343,58 +343,41 @@ components:
...
+          enum:
+            - G1
+            - G2
+            - GA
+            - GB
+            - GB1
+            - GC
+            - FR3_3
+            - FR3_3_GB_G2
+        default_mode:
```
Comment (multun) [inline core/openapi.yaml]: I don't see this field anywhere in RJSRollingStock.java

File: core/openapi.yaml [hunk 5]
```diff
@@ -343,58 +343,41 @@ components:
...
-                format: double
-                minimum: 0
-              max_effort:
-                type: number
-                format: double
-                minimum: 0
-            example:
-              speed: 20
-              max_effort: 400000
-    RollingResistanceDavis:
```
Comment (multun) [inline core/openapi.yaml:366]: please make sure the file is still valid using https://editor.swagger.io/
RollingResistanceDavis cannot be removed while still used

**Review by multun — CHANGES_REQUESTED** [review]
Please compare the spec with the [RJSRollingStock](https://github.com/DGEXSolutions/osrd/blob/43489d5b9fdda5cfa990c9f50593d327e7619b27/core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/rollingstock/RJSRollingStock.java) class in the source code

File: core/openapi.yaml [hunk 6]
```diff
@@ -343,58 +326,58 @@ components:
...
+            type: string
+            example: "protecting systems"
+        power_class:
+          type: string
+          example: "powerClass"
```
Comment (multun) [inline core/openapi.yaml]: `example` should contain an example json value, not the description

**Review by multun — CHANGES_REQUESTED** [review]

File: core/openapi.yaml (hunk 6 above)
Comment (majaziri) [reply core/openapi.yaml]: fixed

**Review by multun — APPROVED** [review]

**Pull request merged**